src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -39,16 +39,12 @@ const useStyles = makeStyles((theme) => ({
 
 function CustomizedInputBase(props) {
   const classes = useStyles();
-  const [state, setState] = React.useState({
-    filter: '',
-  });
-  const { searchText, onChangeHandler } = props;
+  const [selectedFilter, setSelectedFilter] = useState(props.filter);
+  const { searchText, onChangeHandler, onFilterChangeHandler } = props;
 
-  const handleChange = (filter) => (event) => {
-    setState({
-      ...state,
-      [filter]: event.target.value,
-    });
+  const onFilterChange = (event) => {
```

Comment:
This needs to call the function props we've received from above, and pass up the value in our input control.

Reply:
Fixed